keystore/src/entities/test_utils.rs

```diff
@@ -0,0 +1,125 @@
+// Wire
+// Copyright (C) 2022 Wire Swiss GmbH
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+
+use rand::Rng as _;
+
+// ? This is making the tests WAY too slow
+// const MAX_BLOB_SIZE: std::ops::Range<usize> = 1024..MAX_BLOB_LEN;
+const MAX_BLOB_SIZE: std::ops::Range<usize> = 1024..8192;
+
+pub trait EntityTestExt: crate::entities::Entity {
```

Comment:
Can't you move this file in /tests since it looks only used there. Also if possible you could move uuid & rand to dev-dependencies

Reply:
Done